peep.c

```diff
@@ -3032,7 +3032,9 @@ Perl_rpeep(pTHX_ OP *o)
 
             /* Two NEXTSTATEs in a row serve no purpose. Except if they happen
                to carry two labels. For now, take the easier option, and skip
-               this optimisation if the first NEXTSTATE has a label.  */
+               this optimisation if the first NEXTSTATE has a label.
+               XXX: Yves wonders what about if they have different hints or features?
+               */
```

Comment:
The code removes the first COP, not the second, so I think that's safe.

Reply:
Comment changed to note your reply.